frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -109,24 +129,35 @@
             <IndigenousProgrammingAllowance
               v-if="panel.id === INDIGENOUS"
               :indigenousProgrammingModel="getModel(SUPPLEMENTARY_TYPES.INDIGENOUS, nextRenewalTerm)"
-              @update="updateModel"
-              :formDisabled="formDisabled" />
+              :formDisabled="formDisabled"
+              @update="updateModel" />
             <SupportNeedsProgrammingAllowance
               v-if="panel.id === SUPPORT_NEEDS"
               :supportModel="getModel(SUPPLEMENTARY_TYPES.SUPPORT, nextRenewalTerm)"
               :hasInclusionPolicy="currentOrg.hasInclusionPolicy"
               :formDisabled="formDisabled"
               @update="updateModel" />
+          </v-expansion-panel-text>
+        </v-expansion-panel>
+
+        <v-expansion-panel-title class="tooltip py-6 mt-8" hide-actions>
```

Comment:
See previous comment about usage of v-expansion-panel.

Reply:
done